experiments/AMIP/user_io/leaderboard/data_sources.jl

```diff
@@ -0,0 +1,111 @@
+"""
+
+    A struct to describe some observation data that we want to compare against.
+
+"""
+struct ObsDataSource
```

Comment:
Just note that we will compare with data that has a z dimension as well, but we can add that when we are actually doing it.